src/pages/components/accordion/code.mdx

```diff
@@ -9,200 +9,74 @@ tabs: ["Usage", "Style", "Code", "Accessibility"]
-  <AccordionItem title="Panel C">Panel C</AccordionItem>
-</Accordion>
-```
-
-### AccordionItem title
-
-You can use the `title` prop to specify the accordion item's heading. The
-default behavior is to pass in a `string` as the title, however you can also
-pass in a `node`.
-
-_Note: A custom title prop that renders an interactive element is considered an
-accessibility violation. For more information, visit our [reference
-section](#why-is-interactive-content-in-a-heading-considered-an-accessibility-violation) below._
-
-```jsx
-<Accordion align="start">
-  <AccordionItem title={() => <span>Panel A</span>}>Panel A</AccordionItem>
-  <AccordionItem title="Panel B">Panel B</AccordionItem>
-  <AccordionItem title="Panel C">Panel C</AccordionItem>
-</Accordion>
-```
-
-### AccordionSkeleton props
-
-| Prop      | Type                     | Required | Default | Description                                                        |
-| --------- | ------------------------ | -------- | ------- | ------------------------------------------------------------------ |
-| align     | `'start'` &#124; `'end'` | –        | `'end'` | Specify the alignment of the accordion heading title and chevron   |
-| className | `string`                 | –        | –       | Provide an optional className to be applied to the container node  |
-| count     | `boolean`                | –        | `4`     | Specify the number of items to render                              |
-| open      | `boolean`                | –        | `true`  | Specify whether the skeleton should display the first item as open |
-
-Additional props passed into `AccordionSkeleton` will be forwarded along to the
-underlying skeleton container.
-
-### AccordionSkeleton className
-
-The `className` prop passed into `AccordionSkeleton` will be forwarded along to
-the underlying skeleton container. This is useful for specifying a custom class
-name for layout.
-
-```jsx
-<AccordionSkeleton className="custom-class" open count={3} />
-```
-
-## References
-
-<h3 id="why-is-interactive-content-in-a-heading-considered-an-accessibility-violation">
-  Why is interactive content in a heading considered an accessibility violation?
-</h3>
-
-When using the `title` prop from `Accordion`, it is possible to render arbitrary
-markup within the accordion header. The accordion header itself is rendered as a
-`<button>` and is considered an interactive element.
-
-If you render an interactive element inside this header, then it becomes
-nested inside the `<button>`. Common examples of this are buttons or links.
-Rendering interactive content inside the accordion heading is not reachable
-via a keyboard or screen reader.
-
-## Feedback
-
-Help us improve this component by providing feedback, asking questions, and leaving any other comments on [GitHub](https://github.com/carbon-design-system/carbon-website/issues/new?assignees=&labels=feedback&template=feedback.md).
+## Storybooks
```

Comment:
How would you feel about calling it documentation in case folks don't know what storybook is, or just so folks know that's where the docs live?

```suggestion
## Documentation
```

Reply:
Makes sense to me, updated